src/main/java/com/icthh/xm/tmf/ms/activation/config/SecurityConfiguration.java

```diff
@@ -43,12 +43,12 @@ public void configure(HttpSecurity http) throws Exception {
             .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
         .and()
             .authorizeRequests()
-            .antMatchers("/api/**").authenticated()
             .antMatchers("/management/health").permitAll()
             .antMatchers("/management/info").permitAll()
             .antMatchers("/management/prometheus/**").permitAll()
             .antMatchers("/management/**").hasAuthority(RoleConstant.SUPER_ADMIN)
-            .antMatchers("/swagger-resources/configuration/ui").permitAll();
+            .antMatchers("/swagger-resources/configuration/ui").permitAll()
+            .anyRequest().authenticated();
```

Comment:
all endpoints has PreAuthorize annotation

Reply:
added restriction to /DSEntityProvisioning/ endpoint